src/ontology/mondo-edit.obo

```diff
@@ -584370,6 +584369,20 @@ is_a: MONDO:1030005 {source="PMID:31120550"} ! mosaic SMO syndrome
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7768" xsd:anyURI
 
+[Term]
+id: MONDO:1030007
+name: hypertensive urgency
+def: "Hypertensive urgency occurs when blood pressure spikes -- blood pressure readings are 180/110 mmHg or higher -- but there is no damage to the body's organs (such as pulmonary edema, cardiac ischemia, neurologic deficits, or acute renal failure). Blood pressure can be brought down safely within a few hours with blood pressure medication." [icd11.foundation:896211058]
```

Comment:
@twhetzel Can you please make the definition such that it clearly refers to a disease/disorder. For example: "a hypertensive disorder characterized by (blablabla)" 

Please also the PMID for the [publication that Peter shared] (https://www.ncbi.nlm.nih.gov/books/NBK513351/). Some of the information from that publication can also be used for the definition (especially the first sentence)

Reply:
Adding the orcid of the requester as x-ref would be good, but it doesn't look like we have it.

Reply:
I came across the requester's ORCID in another issue, https://orcid.org/0009-0009-6791-365X. I verified this is an actual ORCID, however similar to the GH profile their is no public information available for this ORCID entry.

Reply:
the definition is updated, i will push this change shortly

Reply:
Instead of https://www.ncbi.nlm.nih.gov/books/NBK513351/, please use PMID:30020723